app/util/job.ts

```diff
@@ -56,13 +57,33 @@ export async function completeJob(
     if (!terminalStates.includes(finalStatus)) {
       throw new ConflictError(`Job cannot complete with status of ${finalStatus}.`);
     }
+    const failed = ([JobStatus.FAILED, JobStatus.CANCELED].includes(finalStatus));
```

Comment:
Nit, but I see you copying this logic in metrics.ts. It seems like the logic of which job states correspond to final / done states, which are running states, etc, should be extracted to avoid headaches in adding or altering states.

Reply:
Agreed I was lazy on this one - went ahead and changed it now.